src/middleware/passport/strategies/local/local-guards.ts

```diff
@@ -1,11 +1,6 @@
-import { Request, Response, NextFunction } from 'express';
-import { IUser } from '../../modules/users/model/UserModel';
+import {NextFunction, Request, Response} from "express";
+import {IUser} from "../../../../modules/users/model/UserModel";
 
-/**
- * Middleware function to authorize access based on user roles.
- * @param roles - List of roles that are allowed to access the route.
- * @returns Express middleware function.
- */
 export const authorizeRoles = (...roles: string[]) => {
     return (req: Request, res: Response, next: NextFunction) => {
         // Check if user is authenticated and has one of the specified roles
```

Comment:
Why is this empty?

Reply:
what You mean?

Reply:
the function is empty, and we don't have roles.

Reply:
we have roles.
 role: {
        type: String,
        required: true,
        enum: ['user', 'admin'],
        default: 'user' // Default role
    },
    and we could call if we want

Reply:
yeah, then we should handle them. if the function is empty it does nothing.

Reply:
Ok I see where misunderstanding comes from, unless I open this file in separate tab, I don't see if checks in the function.